x/evm/types/codec.go

```diff
@@ -17,6 +18,6 @@ func init() {
 
 // RegisterCodec registers concrete types and interfaces on the given codec.
 func RegisterCodec(cdc *codec.Codec) {
-	cdc.RegisterConcrete(&EthereumTxMsg{}, "ethermint/MsgEthereumTx", nil)
-	cdc.RegisterConcrete(&EmintMsg{}, "ethermint/MsgEmint", nil)
+	cdc.RegisterConcrete(&EthereumTxMsg{}, appNameLong+"/MsgEthereumTx", nil)
+	cdc.RegisterConcrete(&EmintMsg{}, appNameLong+"/MsgEmint", nil)
```

Comment:
same as previous, I'd say don't change this until you can test this does not break compatibility